lib/fonts/monofonto/read-this.html

```diff
@@ -0,0 +1,191 @@
+<head>

```

Comment:
Not sure exactly how this font is being used but potentially you can add it to `pages/_document.tsx` (I already have the Rubik font in there)

https://nextjs.org/docs/basic-features/font-optimization